manager/integration/tests/test_basic.py

```diff
@@ -3351,6 +3352,10 @@ def test_allow_volume_creation_with_degraded_availability_restore(set_random_bac
         client.by_id_setting(SETTING_REPLICA_NODE_SOFT_ANTI_AFFINITY)
     client.update(replica_soft_anti_affinity_setting, value="false")
 
+    replenish_wait_setting = \
+        client.by_id_setting(SETTING_REPLICA_REPLENISHMENT_WAIT_INTERVAL)
+    client.update(replenish_wait_setting, value="0")
```

Comment:
This reminds me if we will reset all settings after the test case is finished to ensure there will be not any cascading impact for the following test cases? This is more like a general question.

Reply:
Yes, it will get reset, ref [here](https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/common.py#L1378).

Reply:
Do we use that fixture in this case?

Reply:
test_allow_volume_creation_with_degraded_availability_restore(set_random_backupstore, [client](https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/common.py#L1337), core_api, volume_name, csi_pv, pvc, pod, pod_make)

Reply:
ah thanks :)